tests/integration/test_bracken_etl.py

```diff
@@ -24,12 +24,33 @@
 import pytest
 from pandera.errors import SchemaErrors
 
+from taxpasta.application.error import StandardisationError
 from taxpasta.infrastructure.application import (
     BrackenProfileReader,
     BrackenProfileStandardisationService,
 )
 
 
+@pytest.fixture(
+    scope="module",
+    params=[
+        # ("bracken", "2612_pe-ERR5766176_B-db1_S.tsv"),  # noqa: E800
```

Comment:
What does the comment at the end mean (I get conceptually you're checking here taxpasta fails if it tries to merge against other profilers, so don't need to merge against itself)

Reply:
Some of the code linters complain about keeping commented code in the source (linter error E800). I'm disabling that specific complaint on that line with `noqa: E800` because I want to keep it in.